.github/workflows/yarn-build.yml

```diff
@@ -0,0 +1,22 @@
+name: Yarn build and lint
+
+on:
+  pull_request:
+    paths: ['**/*.ts', '**/*.tsx', '**/*.js', '**/*.jsx', '**/*.json', '**/*.md']
```

Comment:
We might want to consider scoping this to the `packages` folder so the yarn workflow doesn't run for changes outside of the `packages` folder. I think right now a change to any `readme.md` would trigger this workflow to run.

Reply:
Good call. I will push a commit to add the `packages/` prefix to each of those paths